Packs/Wiz/IncidentFields/incidentfield-Wiz_Cloud_Account.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I strongly suggest to add a description to all of the incident fields which will describe the meaning of their values.

Reply:
NP. Will add.

Reply:
Added to 58490dc83597062f51291b7ecd158d30fd9f6034